docs/marketplace/jenkins/enterprise-app-store/index.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Setting Up Appcircle Enterprise App Store Plugin For Jenkins
+sidebar_label: Enterprise App Store
+description: Enhance powerful plugin to publish your builds to appcircle app store
+tags:
+  [
+    overview,
+    concepts,
+    app store,
+    internal-testing,
+    beta testing,
+    binary distribution,
+    ipa distribution,
+    apk distribution,
+  ]
+sidebar_position: 2
+---
+
+import Screenshot from '@site/src/components/Screenshot';
+
+# Setting Up Appcircle Enterprise App Store Plugin
+
+The Appcircle Enterprise App Store plugin enables users to publish their apps to the Appcircle App Store.
+
+## System Requirements
+
+**Compatible Agents:**
+
+- macOS
+- Ubuntu
+- Ventura
```

Comment:
- macOS 14.5 also supported. We could update as "macOS 13.x, 14.5". 13.x can be replaced with your Ventura version.
- Ubuntu is not tested.